k-distribution/tests/regression-new/map-symbolic-tests/failing/lookup-9-spec.k.out

```diff
@@ -0,0 +1,3 @@
+<k>
+  lookupResult ( MAP [ Z ] )
+</k>
```

Comment:
same here. i think this should have been passed.

the current term is
```
lookupResult ( MAP [ Y <- 2 ] [ Z ] )
```
which should have been simplified to the expected output, by the rule:
```
rule (MAP:Map [ K1 <- V1 ]) [ K2 ] => MAP [ K2 ] requires K1 =/=K K2
```

Reply:
For another PR.